dask/tests/test_highgraph.py

```diff
@@ -265,3 +265,18 @@ def test_len_does_not_materialize():
     assert len(hg) == len(a) + len(b) == 7
 
     assert not hg.layers["b"].is_materialized()
+
+
+def test_node_tooltips_exist():
+    da = pytest.importorskip("dask.array")
```

Comment:
You will also need to skip this test if graphviz is not installed. The mindeps-array-dataframe CI test is failing, and I think that is because graphviz is not included [in the environment](https://github.com/dask/dask/blob/main/continuous_integration/environment-mindeps-array-dataframe.yaml). Adding another line with pytest importorskip should fix that problem.

(There is another failing CI check, but I don't think that is related to anything you've been working on here. It might magically become good next time it runs,  we'll see)